marklogic-data-hub-central/ui/e2e/cypress/fixtures/users/developer.json

```diff
@@ -2,5 +2,5 @@
     "user-name": "dh-dev",
     "description": "A data hub dev user",
     "password": "dh-dev",
-    "role": ["data-hub-developer", "hub-central-entity-exporter", "hub-central-mapping-reader", "hub-central-mapping-writer"]
+    "role": ["data-hub-developer", "hub-central-entity-exporter", "hub-central-load-reader", "hub-central-mapping-reader", "hub-central-mapping-writer"]
```

Comment:
Cypress tests only test explorer for now, so its ok to not update developer and operator roles. There also is an open PR with which Cypress tests can run with any roles, just like we set roles to a user in core HC tests.

Reply:
Thanks. Good to know. I've removed the unnecessary roles with my latest PR updates.